plugin/showpass-wordpress-plugin-shortcode.php

```diff
@@ -815,16 +829,8 @@ function showpass_widget_expand($atts, $content = null) {
 			$include_icon = true;
 		}
 
-		if ((isset($atts['keep_shopping']) && $atts['keep_shopping'] === 'true') || (get_option('option_keep_shopping') === 'false')) {
-			$keep_shopping = 'true';
-		} else {
-			$keep_shopping = 'false';
-		}
-
-		if ((isset($atts['keep_shopping']) && $atts['keep_shopping'] === 'false') || (get_option('option_keep_shopping') != 'false')) {
-			$keep_shopping = 'false';
-		} else {
-			$keep_shopping = 'true';
+		if (isset($atts['keep_shopping'])) {
```

Comment:
What was `option_keep_shopping` for before?

Reply:
Not working